src/spaceone/inventory_v2/service/namespace_service.py

```diff
@@ -0,0 +1,235 @@
+import logging
+from typing import Union
+
+from pkg_resources import working_set
+from spaceone.core.service import *
+from spaceone.core.service.utils import *
+from spaceone.core.error import *
+
+from spaceone.inventory_v2.model.namespace.request import *
+from spaceone.inventory_v2.model.namespace.response import *
+from spaceone.inventory_v2.manager.namespace_manager import NamespaceManager
+from spaceone.inventory_v2.manager.identity_manager import IdentityManager
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@authentication_handler
+@authorization_handler
+@mutation_handler
+@event_handler
+class NamespaceService(BaseService):
+    resource = "Namespace"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.namespace_mgr = NamespaceManager()
+        self.identity_mgr = IdentityManager()
+
+    @transaction(
+        permission="inventory-v2:Namespace.write",
+        role_types=["DOMAIN_ADMIN", "WORKSPACE_OWNER"],
+    )
+    @convert_model
+    def create(self, params: NamespaceCreateRequest) -> Union[NamespaceResponse, dict]:
+        """Create namespace
+
+        Args:
+            params (dict): {
+                'namespace_id': 'str',
+                'name': 'str',                  # required
+                'category': 'str',              # required
+                'icon': 'str',
+                'tags': 'dict',
+                'resource_group': 'str',        # required
+                'namespace_group_id': 'str',    # required
+                'workspace_id': 'str',          # injected from auth
+                'domain_id': 'str',             # injected from auth (required)
+            }
+
+        Returns:
+            NamespaceResponse:
+        """
+
+        domain_id = params.domain_id
+        workspace_id = params.workspace_id
+        resource_group = params.resource_group
+
+        # Check permission by resource group
+        if resource_group == "WORKSPACE":
+            if workspace_id is None:
+                raise ERROR_REQUIRED_PARAMETER(key="workspace_id")
+
+            self.identity_mgr.check_workspace(workspace_id, domain_id)
+        else:
+            params.workspace_id = "*"
+
```

Comment:
Before create namespace, i think we have to check `namespace_group` object with `namespace_group_id` value at params.